### PR TITLE
Simplify openshift-ci

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

**Comment by rnc** [comment]
@stuartwdouglas Do any of the tests trigger the redhat-appstudio-jvm-build-service-main.yaml (i.e. the openshift-ci) so I can verify any changes to it still work?

File: deploy/openshift-ci.sh [hunk 1]
```diff
@@ -12,9 +12,26 @@ echo ${JVM_BUILD_SERVICE_CACHE_IMAGE}
...
+        if [ $(date +%s) -gt $endTime ]; then
+            echo "Unable to find $1"
+            exit 1
+        fi
+    done
+}
+
 DIR=`dirname $0`
 echo "Running out of ${DIR}"
-$DIR/install-openshift-pipelines.sh
```
Comment (stuartwdouglas) [inline deploy/openshift-ci.sh:17]: This is also useful when setting up a new hypershift or clusterbot cluster

Comment (stuartwdouglas) [reply deploy/openshift-ci.sh:17]: Actually I guess it does not really matter.

**Pull request merged**